tools/secret-management/Directory.Build.props

```
@@ -6,7 +6,7 @@
         <IsPackable>false</IsPackable>
         <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-        <WarningsNotAsErrors>8002</WarningsNotAsErrors>
+        <NoWarn>CS8002</NoWarn>
```

```suggestion
        <NoWarn>$(NoWarn);CS8002</NoWarn>
```

However as you pinged offline this will still cause signing of the library to fail so we should see if we can find another way to do this with a strong name signed assembly.